railties/lib/rails/commands/test/test_command.rb

```diff
@@ -1,21 +1,41 @@
 require "rails/command"
-require "rails/test_unit/minitest_plugin"
+require "rails/test_unit/runner"
 
 module Rails
   module Command
     class TestCommand < Base # :nodoc:
       no_commands do
         def help
```

Comment:
I noticed that but opted to separate them out. If we're running with `ruby -Itest test/models --help` we aren't running directly through the Rails test command, so I liked that we're slightly indicating that in the different help output. For instance when tests are run with `ruby` we can't support the `-e` option (iirc the app is booted once `rails/test_help` is run, so invoking the test command from there won't work either).